samples/dashboards/cypress/component-datagrid/demo.mjs

```diff
@@ -82,6 +83,7 @@ Dashboards.board('container', {
             editable: true,
             sync: {
                 highlight: true,
+                visibility: true,
```

Comment:
In HC we use `visible` so I would expect to have the same name here.

Reply:
Everywhere in Dash, we use `visibility`, @pawelfus should we change that? 
IMO the `visible` might be confusing because it looks like the `sync` might or might not be visible.